installation/resources/crds/telemetry/logpipelines.crd.yaml

```diff
@@ -341,7 +341,7 @@ spec:
                 type: array
               unsupportedMode:
                 description: Is active when the LogPipeline uses a `custom` output
-                  or filter; see [unsupported mode](./../../01-overview/main-areas/telemetry/telemetry-02-logs.md#unsupported-mode#unsupported-mode).
+                  or filter; see [unsupported mode](./../../01-overview/telemetry/telemetry-02-logs.md#unsupported-mode#unsupported-mode).
```

Comment:
We should not use relative links in the CRD description since this is also used outside of the docs and will not work there. Can you fix this within this PR or should we create a follow-up PR?

Reply:
> We should not use relative links in the CRD description since this is also used outside of the docs and will not work there. Can you fix this within this PR or should we create a follow-up PR?

TBH, I'd prefer to do it in another PR.